### PR TITLE
Feat: app layout

### PR DESCRIPTION
**Pull request by jennifer-hoang**
Setup initial layout

### CODE REVIEW

**Comment by jennifer-hoang** [comment]
Hi @christopheralex ,
I realized I'm having the same problem that we had previously where resizing alters the app layout. Do you remember how you fixed it previously? Was it by specifying the plot sizes for each graph?

**Comment by christopheralex** [comment]
> Hi @christopheralex , I realized I'm having the same problem that we had previously where resizing alters the app layout. Do you remember how you fixed it previously? Was it by specifying the plot sizes for each graph?

Hey Jenn, we might have to manually change the values of my plot sizes. Thats how we solved it last time.

File: app.R [hunk 1]
```diff
@@ -1,4 +1,345 @@
...
+library(dashCoreComponents)
+library(ggplot2)
+library(plotly)
+library(purrr)
+library(readr)
+library(dplyr)
+
+# Read data ------
+
+df <- read_csv("https://raw.githubusercontent.com/rfordatascience/tidytuesday/master/data/2020/2020-01-21/spotify_songs.csv")
```
Comment (christopheralex) [inline app.R]: Could change it to local data. As URL request from heroku will again take time.

**Comment by christopheralex** [comment]
I am not sure if this was for me to do or you just missed it . 
Only my plot is missing 😂 . 
I had given my Individual assignment here - https://github.com/UBC-MDS/spotify-explorer-R/issues/5#issuecomment-1068600762
https://github.com/christopheralex/dash-r-heroku-demo/blob/main/app.R
Either way its cool if I have to add it .

File: app.R [hunk 2]
```diff
@@ -1,4 +1,345 @@
-# author: Christopher Alexander, Jennifer Hoang, Michelle Wang, Thea Wenxin
-# date: 2022-03-01
+# author: Christopher Alexander, Jennifer Hoang, Michelle Wang, Wenxin Xiang
+# date: 2022-03-15
 
-pass
+library(dash)
+library(dashHtmlComponents)
+library(dashBootstrapComponents)
```
Comment (christopheralex) [inline app.R]: Not sure why , but I am not able to install this package `dashBootstrapComponents` in my local . I only need `dashHtmlComponents` and `dash` to run the code .

**Comment by christopheralex** [comment]
You could also create a file called `spotify-r-heroku.Rproj` to make an R project file which helps in creating a new environment.
Below is the code for the file
```
Version: 1.0

RestoreWorkspace: Default
SaveWorkspace: Default
AlwaysSaveHistory: Default

EnableCodeIndexing: Yes
UseSpacesForTab: Yes
NumSpacesForTab: 4
Encoding: UTF-8

RnwWeave: Sweave
LaTeX: pdfLaTeX
```

**Comment by jennifer-hoang** [comment]
Thanks Chris, I've made the following edits
- Removed dashHtmlComponents, dashBootstrapComponents
- Change to read data locally
- Rproj file was added

Current issues:
@xiangwxt @christopheralex I'm having some trouble getting your plots to render. Would you be able to double check to see if I've correctly added in your plots? I've temporarily commented them out for now.

I suggest we merge and then work on it.

File: app.R (hunk 1 above)
Comment (xiangwxt) [reply app.R]: I will need to test it out on my local, last time it cannot read data from local directory.

**Comment by xiangwxt** [comment]
> You could also create a file called `spotify-r-heroku.Rproj` to make an R project file which helps in creating a new environment. Below is the code for the file
> 
> ```
> Version: 1.0
> 
> RestoreWorkspace: Default
> SaveWorkspace: Default
> AlwaysSaveHistory: Default
> 
> EnableCodeIndexing: Yes
> UseSpacesForTab: Yes
> NumSpacesForTab: 4
> Encoding: UTF-8
> 
> RnwWeave: Sweave
> LaTeX: pdfLaTeX
> ```

It's a R project now.

File: app.R [hunk 3]
```diff
@@ -1,4 +1,359 @@
-# author: Christopher Alexander, Jennifer Hoang, Michelle Wang, Thea Wenxin
-# date: 2022-03-01
+# author: Christopher Alexander, Jennifer Hoang, Michelle Wang, Wenxin Xiang
+# date: 2022-03-15
 
-pass
+library(dash)
+library(dashHtmlComponents)
```
Comment (xiangwxt) [inline app.R:5]:
```suggestion
library(dashHtmlComponents)
library(dashCoreComponents)
library(dashBootstrapComponents)
```

Comment (christopheralex) [reply app.R:5]: I dont think we need dashCoreComponents as it is in dashHtmlComponents or dash . Thats what Florencia told I guess.

Comment (xiangwxt) [reply app.R:5]: I think we used these two libraries in the code.

Comment (christopheralex) [reply app.R:5]: Well `dashBootstrapComponents` does not work for me, I cant even install it . Can you try running it without dashBootstrapComponents and let me know if it works.  If it does , lets skip .

Comment (xiangwxt) [reply app.R:5]: Ok, let's try.

**Review by xiangwxt — APPROVED** [review]

**Pull request merged**